core/templates/pages/exploration-editor-page/history-tab/history-tab.component.ts

```diff
@@ -249,6 +249,20 @@ export class HistoryTabComponent
     }
   }
 
+
+  isDropdownVisible = false;
+
+  moreOptionButton(event: KeyboardEvent, dropdownButton: HTMLElement): void {
+    if (event.key === 'Enter') {
+      const dropdownButtonId = dropdownButton.id;
+      const index = dropdownButtonId.substring(
+        dropdownButtonId.lastIndexOf('-') + 1);
+      const dropdownMenuButton = document.getElementById(
+        `dropdownMenuButton-${index}`);
+      dropdownMenuButton?.click();
```

Comment:
Is there another way to show more options that does not involve programmatically "clicking" / interacting with the DOM? In general, we try to avoid triggering click events (or other user actions/events) programmatically in the codebase.